build.sbt

```diff
@@ -10,37 +10,31 @@ lazy val runtime = (project in file("runtime"))
     crossScalaVersions := Seq("2.12.4", "2.11.11"),
     name := "GrpcGatewayRuntime",
     libraryDependencies ++= Seq(
-      "com.trueaccord.scalapb" %% "compilerplugin"          % "0.6.7",
-      "com.trueaccord.scalapb" %% "scalapb-runtime-grpc"    % "0.6.7",
       "com.trueaccord.scalapb" %% "scalapb-runtime-grpc"    % "0.6.7",
       "com.trueaccord.scalapb" %% "scalapb-json4s"          % "0.3.3",
       "io.grpc"                %  "grpc-netty"              % "1.8.0",
       "org.webjars"            %  "swagger-ui"              % "3.5.0",
-      "com.google.api.grpc"    % "googleapis-common-protos" % "0.0.3" % "protobuf"
-    ),
-    PB.protoSources in Compile += target.value / "protobuf_external",
-    includeFilter in PB.generate := new SimpleFilter(
-      file => file.endsWith("annotations.proto") || file.endsWith("http.proto")
-    ),
-    PB.targets in Compile += scalapb.gen() -> (sourceManaged in Compile).value,
-    mappings in (Compile, packageBin) ++= Seq(
-      baseDirectory.value / "target" / "protobuf_external" / "google" / "api" / "annotations.proto" -> "google/api/annotations.proto",
-      baseDirectory.value / "target" / "protobuf_external" / "google" / "api" / "http.proto"        -> "google/api/http.proto"
     )
   )
 
 lazy val generator = (project in file("generator"))
+  .enablePlugins(BuildInfoPlugin)
   .settings(
+    buildInfoPackage := "grpcgateway.generators",
     crossScalaVersions := Seq("2.12.4", "2.10.6"),
     name := "GrpcGatewayGenerator",
     libraryDependencies ++= Seq(
       "com.trueaccord.scalapb" %% "compilerplugin"          % "0.6.7",
-      "com.trueaccord.scalapb" %% "scalapb-runtime-grpc"    % "0.6.7",
-      "com.google.api.grpc"    % "googleapis-common-protos" % "0.0.3" % "protobuf"
```

Comment:
why do we depend on scalapb-runtime-grpc? We don't need this, because it's just source code generator

Reply:
In theory it's not needed but I think there is a compile error if missing

Reply:
I belive user has to specify this dependency "googleapis-common-protos" explicitly, no need to depend on this in generator. I don't have any compile errors.
"generator" artifact has to depend only on libraries that it needs. There's only one dependency "compilerplugin"